package.json

```diff
@@ -20,6 +20,7 @@
     "express-handlebars": "^3.0.0",
     "fluent": "^0.8.1",
     "fluent-langneg": "^0.1.0",
+    "full-icu": "^1.2.1",
```

Comment:
This installs all the icu data that `Intl` needs to format dates & numbers with long-form months, etc.

Reply:
See https://nodejs.org/api/intl.html#intl_embed_the_entire_icu_full_icu